include/simdutf/encoding_types.h

```diff
@@ -17,7 +15,7 @@ enum endianness {
         BIG
 };
 
-std::string to_string(encoding_type bom);
+//std::string to_string(encoding_type bom);
```

Comment:
@lemire I think the builds are failing because of this.

Reply:
Oh.

Reply:
Fixed.